global.json

```diff
@@ -1,6 +1,6 @@
 {
     "projects": [ "src" ],
     "sdk": {
-        "version": "1.0.0-preview2-003121"
+        "version": "1.0.4"
```

Comment:
I also pegged this at the 2.0.0 tools and it works fine if we decide to go with that.